pkgs/top-level/aliases.nix

```diff
@@ -954,6 +954,7 @@ mapAliases ({
   tokodon = plasma5Packages.tokodon;
   tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
   transfig = fig2dev; # Added 2022-02-15
+  transifex-client = transifex-cli; # Added 2023-12-29
```

Comment:
Since this is probably not a drop-in replacement, we should use a throw.

Reply:
Actually it's drop-in replacement.

Reply:
Thanks for clearing up